Helper_Scripts/generate_audio_bundle_docs.py

```
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+"""Generate bundle-driven audio setup documentation snippets."""
+
+from __future__ import annotations
+
+from typing import Iterable
```

Ruff (0.15.6)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)